### PR TITLE
luke/DisplayInput - createClass Conversion

### PR DESCRIPTION
**Pull request by lukeschunk**
Convers `DisplayInput.js` and `DisplayInputDocs.js` to Class extends syntax.

### CODE REVIEW

**Comment by paniclater** [comment]
:shipit:

File: src/components/DisplayInput.js [hunk 1]
```diff
@@ -33,11 +35,16 @@ class DisplayInput extends React.Component {
     valid: true
   };
 
-  _isLargeOrMediumWindowSize = () => {
+  componentWillMount () {
+    this._labelId = _uniqueId('DI');
+    this._inputId = _uniqueId('DI');
```
Comment (paniclater) [inline src/components/DisplayInput.js]: what's the 'DI' here?

Comment (lukeschunk) [inline src/components/DisplayInput.js]: this is used to associate the label with the input, introduced here  https://github.com/mxenabled/mx-react-components/pull/559

Comment (jtanner) [inline src/components/DisplayInput.js]: It's short for `DisplayInput`. Do you think we should change it or add a comment?

Comment (paniclater) [inline src/components/DisplayInput.js]: nope that's ok just curious.

**Comment by paniclater** [comment]
:shipit:

**Comment by lukeschunk** [comment]
closing this as this will be covered by https://github.com/mxenabled/mx-react-components/pull/565 and https://github.com/mxenabled/mx-react-components/pull/566

**Pull request closed without merging**